src/musig.rs

```diff
@@ -725,6 +726,7 @@ impl<T: SigningTranscript+Clone> MuSig<T,CollectStage> {
         let reveal = their_reveal.into_points() ?;
         let s = Scalar::from_canonical_bytes(their_cosignature.0)
             .ok_or(SignatureError::ScalarFormatError) ?;
+        let s = Option::<Scalar>::from(s).ok_or(SignatureError::ScalarFormatError)?;
```

Comment:
:question: similar to above, IFF this file needs to change
```suggestion
        let s = Scalar::from_canonical_bytes(their_cosignature.0);
        let s = Option::<Scalar>::from(s).ok_or(SignatureError::ScalarFormatError)?;
```

Reply:
Done.